### PR TITLE
Update Argentinian conversion and tax rates

### PR DESCRIPTION
**Pull request by LouiseBonnard**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Update localization packs data, update the conversion rate and add two extra tax rate - feedback from the Argentinian ambassador ([here](https://docs.google.com/spreadsheets/d/1QvBo1q_ddSZU3wcawGN_oZVeutY-CB5eJaYfky6HXys/edit?usp=sharing)) and partner agency ([here](https://docs.google.com/spreadsheets/d/1_7K8p_wif-p1rs5xGXx6_p3NrY0TuLjgYPa-sNFzQKk/edit?usp=sharing))
| Type?         | improvement
| Category?     | LO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | It is part of this [EPIC](https://github.com/PrestaShop/PrestaShop/issues/15829).

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16078)

### CODE REVIEW

File: localization/ar.xml [hunk 1]
```diff
@@ -18,6 +20,14 @@
 		<taxRulesGroup name="AR Reduced rate (10.5%)">
             <taxRule iso_code_country="ar" id_tax="2" />
         </taxRulesGroup>
+    
+    <taxRulesGroup name="AR Reduced rate (2.5%)">
+            <taxRule iso_code_country="ar" id_tax="3" />
+        </taxRulesGroup>
+    
+        <taxRulesGroup name="AR Zero rate (0%)">
```
Comment (Progi1984) [inline localization/ar.xml]:
```suggestion
    <taxRulesGroup name="AR Zero rate (0%)">
```

**Review by Progi1984 — DISMISSED** [review]

**Review by eternoendless — DISMISSED** [review]

**Comment by eternoendless** [comment]
Branch must be deleted after merge

**Comment by Progi1984** [comment]
@LouiseBonnard You have a conflict on this PR.

**Comment by LouiseBonnard** [comment]
@Progi1984, thanks! I don't have a clue why there is a conflict here... could it be because two taxes (AR Reduced Rate) share the same name?
🤔

**Comment by eternoendless** [comment]
Rebased

**Review by eternoendless — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @LouiseBonnard

**Pull request merged**